app/src/main/java/edu/uc/kansakpb/ikeahjalpare3/ui/main/MainFragment.kt

```diff
@@ -30,8 +37,57 @@ class MainFragment : Fragment() {
         super.onActivityCreated(savedInstanceState)
         viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
         viewModel.furnitures.observe(viewLifecycleOwner, Observer {
-                _furnitures: ArrayList<Furniture> -> actFurnitureName.setAdapter(ArrayAdapter(context!!, R.layout.support_simple_spinner_dropdown_item, _furnitures))
+
+                _furnitures -> actFurnitureName.setAdapter(ArrayAdapter(context!!, R.layout.support_simple_spinner_dropdown_item, _furnitures))
+
         })
+
+        actFurnitureName.onItemSelectedListener =  object : AdapterView.OnItemSelectedListener {
```

Comment:
All of the empty methods indicate incomplete functionality.  Only complete features, which meet the team's Definition of Done, should be merged to master.  That counts for the code review branch, too.  If this feature is not complete, I would not merge this pull request.